### PR TITLE
Add signin + copy to anon package view

### PR DESCRIPTION
**Pull request by whit537**
Implements first mock from https://github.com/gratipay/gratipay.com/issues/4297#issuecomment-273854209.

----------------------

screen shot 2017-01-19 at 1 14 43 pm

----------------------

screen shot 2017-01-24 at 5 40 10 pm

### CODE REVIEW

**Comment by whit537** [comment]
Ready for review, @JessaWitzel @mattbk @nobodxbodon.

**Comment by whit537** [comment]
Note that the base is #4305 and not `master`, so we should be good to merge even though the final state is incomplete.

File: www/on/npm/%package/index.html.spt [hunk 1]
```diff
@@ -27,4 +27,15 @@ url = 'https://npmjs.com/package/' + package.name
 {% endblock %}
 
 {% block content %}
+{% if user.ANON %}
+<h2>{{ _( '{npm_package} has not been claimed on Gratipay.'
+        , npm_package='npm/' + package_name
```
Comment (nobodxbodon) [inline www/on/npm/%package/index.html.spt]: Seems it's not highlighted as link in screenshot. But I don't want to block us on this.

Comment (mattbk) [inline www/on/npm/%package/index.html.spt]: Is that because it's <h2>?

Comment (whit537) [inline www/on/npm/%package/index.html.spt]: Ah, good call! Fixed in #4307.

**Pull request merged**